gradle.properties

```
@@ -1,4 +1,4 @@
 // Project properties
-version=2.4.0
+version=2.4.1
```

if you think its a new feature then update this too
```suggestion
version=2.5.0
```